### DISCUSSION TITLE
What is the latest WDIO version i can use with WDI5?

### DISCUSSION THREAD

**Discussion by smrtyviks** [ui5-community/wdi5#540] [Q&A] [answered]
Hi All, 

I recently updated my WDIO version to latest 8.16.3 and the WDI5 script are not working anymore.

Can you please let me know what is the latest version of WDIO that is compatible with WDI5?

Regards,
Vikrant

**Comment by vobu** [comment, marked as answer]
wdi5^1 is wdio^7 compatible. but wdi5^2 will be released either this or next week and it will be fully wdio^8 compatible

**Reply by Vikrant87-lego** [reply]
Thank you for the response

**Reply by vobu** [reply]
wdi5 2.0 is out now, incl full wdio^8 compat https://blogs.sap.com/2023/09/16/wdi5-2.0-released/
